.github/workflows/e2e-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.16'
-      - run: make e2e-local E2E_NODES=2 JUNIT_DIRECTORY=./artifacts/
+      - run: make e2e-local E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
```

Comment:
Note: we'd need to also update the downstream o/release configuration for this environment variable as well: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-release-4.10.yaml#L81.